Include/Rover/TrialView.hpp

```diff
@@ -110,16 +110,16 @@ namespace Rover {
 
     private:
       std::size_t m_size;
-      GetPtr m_get_ptr;
+      Get m_get;
```

Comment:
Types should use a noun as a name, not a verb. Rename this to a more descriptive noun.